config/custom-environment-variables.json

```diff
@@ -0,0 +1,15 @@
+{
+  "knex": {
+    "connection": {
+      "host": "PSQL_HOST",
+      "user": "PSQL_USER",
+      "port": "PSQL_PORT",
+      "password": "PSQL_PASSWORD",
+      "database": "PSQL_DATABASE"
+    },
+    "pool": {
+      "min": "PSQL_POOL_MIN",
+      "max": "PSQL_POOL_MAX"
```

Comment:
@vgeorge we're getting an error because knex is looking for an int here but it's seeing a string. Is there any way to force pool.min and pool.max to be integers? I guess all env vars come across as strings, but this wasn't an issue previously. I tried editing knexfile.js and manually changing them to be int after they were imported but it looks like the config object we get back is immutable, so didn't know if there was a better way to do this.